### PR TITLE
[Issue-2380] Add script for issue 2380 [internal review]

### PR DESCRIPTION
**Pull request by VjKlepikov**
ATF Test Scripts to check [FORDTCN-12049](https://adc.luxoft.com/jira/browse/FORDTCN-12049)

This PR is **[ready]** for review.

### Summary
New ATF scripts to check the issue.

### ATF version
develop

### Changelog
Added new scripts
Updated test set for the issues in scope of 7.2 release

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/Defects/7_2/2380_PoliciesManager_must_allow_all_requested_VD_params_in_case_parameters_field_is_omitted.lua [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    rpcs = {
+      [common.rpc.sub] = {
+        hmi_levels = { "NONE", "BACKGROUND", "FULL", "LIMITED" },
+      },
+      [common.rpc.on] = {
+        hmi_levels = { "NONE", "BACKGROUND", "FULL", "LIMITED" },
+      },
+      [common.rpc.unsub] = {
+        hmi_levels = { "NONE", "BACKGROUND", "FULL", "LIMITED" },
+      }
```
Comment (dboltovskyi) [inline test_scripts/Defects/7_2/2380_PoliciesManager_must_allow_all_requested_VD_params_in_case_parameters_field_is_omitted.lua:48]: I would explicitly define for each RPC that `parameters` field is omitted. E.g.:
```
      [common.rpc.sub] = {
            parameters = nil,
            hmi_levels = { "NONE", "BACKGROUND", "FULL", "LIMITED" },
}


```

File: test_sets/Defects/Defects_release_7_2.txt [hunk 2]
```diff
@@ -2,3 +2,4 @@
 ./test_scripts/Defects/7_2/3556_2_GetListOfPermission_contains_groups_with_user_consent_prompt.lua
 ./test_scripts/Defects/7_2/977_SDL_sends_appName_in_vrSynonyms_and_ttsName_in_case_of_lower_and_upper_bound_values_of_params_in_json_file.lua
 ./test_scripts/Defects/7_2/2398_SendLocation_with_one_OR_more_allowed_params_and_one_OR_more_disallowed_params_by_user.lua
+./test_scripts/Defects/7_2/2380_PoliciesManager_must_allow_all_requested_VD_params_in_case_parameters_field_is_omitted.lua
```
Comment (dboltovskyi) [inline test_sets/Defects/Defects_release_7_2.txt:5]: Propose to make this new script to be part of VD test set (`./test_sets/API/vehicle_data.txt`) and move it to `test_scripts/API/VehicleData/OnVehicleData` folder

**Review by dboltovskyi — CHANGES_REQUESTED** [review]

**Comment by VjKlepikov** [comment]
@dboltovskyi Thank you for the review, all comments have been processed [c9fdadd](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/115/commits/c9fdadd1fcf9fe2ce94837c56af2fb35318ea6f1)

File: test_sets/API/vehicle_data.txt [hunk 3]
```diff
@@ -16,6 +16,7 @@
 ./test_scripts/API/VehicleData/OnVehicleData/006_OnVD_mandatory_parameters.lua
 ./test_scripts/API/VehicleData/OnVehicleData/007_OnVD_min_max_boundary_values.lua
 ./test_scripts/API/VehicleData/OnVehicleData/008_OnVD_enum_and_bool_values.lua
+./test_scripts/API/VehicleData/OnVehicleData/009_PoliciesManager_must_allow_all_requested_VD_params_in_case_parameters_field_is_omitted.lua
```
Comment (dboltovskyi) [inline test_sets/API/vehicle_data.txt:19]: Let's rename this script to conform other ones for VD. E.g. `009_OnVD_allowed_for_omitted_parameters_field.lua`

**Review by dboltovskyi — APPROVED** [review]

File: test_sets/API/vehicle_data.txt (hunk 3 above)
Comment (VjKlepikov) [reply test_sets/API/vehicle_data.txt]: @dboltovskyi done, thanks

**Pull request merged**